demo/demo-jaxrs/jaxrs-server/src/main/java/org/apache/servicecomb/demo/jaxrs/server/JaxRSDefaultValues.java

```diff
@@ -88,4 +88,34 @@ public String queryPackages(HttpServletRequest httpRequest,
     return "" + appType;
   }
 
+  @Path("/javaprimitive1")
+  @POST
+  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+  public String jaxRsJavaPrim1(@FormParam("a") int a, @DefaultValue("bobo") @FormParam("b") String b) {
```

Comment:
test case for different parameter type, do not named them 1/2/3/4......

Reply:
Fixed